### ISSUE TITLE
Where is the testcsv.csv in py code

### ISSUE THREAD

**Issue by xiaohui96** [kimmyungsup/PV-forecasting-tensorflow#1]
Hi, can upload the testcsv.csv file? i cannot find in this depository ...

**Comment by kimmyungsup** [comment]
Sorry. I just had changed the file name to "pv_test.csv" in the code. It is the same file as that one.

**Comment by xiaohui96** [comment]
Yeah, i have seen your changes. Have the data in pv_test.csv all coming from the uploaded csv named pv_data.csv (or pv_data.csv)? Could you please directly upload the file pv_test.csv , thanks...

**Comment by kimmyungsup** [comment]
Yes it does. Both are the same files.

**Comment by xiaohui96** [comment]
OK, i got it. thanks...
